samcli/commands/local/lib/local_api_service.py

```diff
@@ -34,9 +34,7 @@ def __init__(self, lambda_invoke_context, port, host, static_dir):
         self.static_dir = static_dir
 
         self.cwd = lambda_invoke_context.get_cwd()
-        self.api_provider = ApiProvider(
-            lambda_invoke_context.template, parameter_overrides=lambda_invoke_context.parameter_overrides, cwd=self.cwd
```

Comment:
why did we remove this `parameter_overrides`?

Reply:
`Stack` object encapsulates `template` and `parameter_overrides`